VxWorks 6.9 does not provide `uintmax_t` type

**Describe the bug**
A new compiler error has appeared on the vxworks-6.9 build:
```
/home/jphickey/cfecfs/github/osal/ut_assert/src/utstubs.c: In function 'UT_Hook_GetArgPtr':
/home/jphickey/cfecfs/github/osal/ut_assert/src/utstubs.c:602: error: expected specifier-qualifier-list before 'uintmax_t'
```

**To Reproduce**
Build using the VxWorks 6.9 toolchain

**Expected behavior**
Should build successfully

**System observed on:**
VxWorks 6.9, gcc 4.3.3 on gs582w-cfelnx lab machine.

**Reporter Info**
Joseph Hickey, Vantage Systems, Inc.

--- a/ut_assert/src/utstubs.c
+++ b/ut_assert/src/utstubs.c
@@ -599,7 +599,7 @@ const void* UT_Hook_GetArgPtr(const UT_StubContext_t *ContextPtr, const char *Na
 
     static const union
     {
-        uintmax_t AsInt;
+        unsigned long AsInt;
         void *AsPtr;
         double AsFloat;
     } ARG_DEFAULT_ZERO_VALUE = { 0 };

